semgrep/tests/e2e/snapshots/test_output/test_sarif_output_include_nosemgrep/results.sarif

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
+  "fixes": [],
```

Comment:
Can you add a test that actually contains a fix object?

Reply:
Done in e8a3ad7db5c64827c160b7d83ca52aaf6a367211